docs/content/integrations/dbt/using-dbt-with-dagster/load-dbt-models.mdx

```diff
@@ -15,113 +15,146 @@ description: Dagster can orchestrate dbt alongside other technologies.
 
 At this point, you should have a [fully-configured dbt project](/integrations/dbt/using-dbt-with-dagster/set-up-dbt-project) that's ready to work with Dagster.
 
-In this step, you'll finally begin integrating dbt with Dagster! To do this, you'll:
+In this section, you'll finally begin integrating dbt with Dagster! To do this, you'll:
 
-- [Load the dbt models into Dagster as assets](#step-1-load-the-dbt-models-as-assets)
-- [Define a Dagster code location](#step-2-define-a-dagster-code-location)
-- [View the assets in the Dagster UI](#step-3-view-the-assets-in-the-dagster-ui)
+- [Install Dagster and the Dagster-dbt library](#step-0-install-dagster-and-the-dagster-dbt-library)
+- [Create a Dagster project that wraps your dbt project](#step-1-create-a-dagster-project-that-wraps-your-dbt-project)
+- [Inspect your Dagster project in Dagster's UI](#step-2-inspect-your-dagster-project-in-dagsters-ui)
+- [Build your dbt project, aka materialize your assets](#step-3-build-your-dbt-project-aka-materialize-your-assets)
+- [Review the Python code in your Dagster project](#step-4-review-the-python-code-in-your-dagster-project)
 
 ---
 
-## Step 1: Load the dbt models as assets
+## Step 0: Install Dagster and the Dagster-dbt library
```

Comment:
This should be step 1. I recognize it's a pain to update the other headings + links (at least right now), but I'm talking to Dish about infra improvements that would eliminate the need to do this in the future.

Reply:
Not so bad to change them!